### ISSUE TITLE
Documentation for averaging logged values when using accumulate_grad_batches

### ISSUE THREAD

**Issue by cajoek** [Lightning-AI/lightning#14301]
## 📚 Documentation
It would be very nice if in the [documentation](https://pytorch-lightning.readthedocs.io/en/stable/advanced/training_tricks.html#accumulate-gradients) there was a suggestion of how to average the logged values in one accumulation group. I'm not sure what the best way is to do this.

**Comment by awaelchli** [comment]
Honestly, I'm also not sure how I would do it. It is a bit difficult to access this information from within the LightningModule. So I agree it would be good to show the best practice for this in the docs.

**Comment by rohitgr7** [comment]
one can just capture the logs and use average meter to capture the accumulated one and log it accordingly.
```py
loss = compute_loss()
self.acc_loss += loss

if batch_idx % self.trainer.accumulate_grad_batches == 0:
    self.acc_loss = self.acc_loss / self.trainer.accumulate_grad_batches
    self.log(...)
    self.acc_loss = 0
```
